src/frontend/src/pages/InfoPage.tsx

```diff
@@ -3,179 +3,159 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Col, Container, Row } from 'react-bootstrap';
-import {
-  faScroll,
-  faCode,
-  faCommentAlt,
-  faBolt,
-  faCog,
-  faDollarSign,
-  faSearch
-} from '@fortawesome/free-solid-svg-icons';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import { faScroll, faCode, faCommentAlt, faBolt, faCog, faDollarSign, faSearch } from '@fortawesome/free-solid-svg-icons';
 import ExternalLink from '../components/ExternalLink';
 import PageTitle from '../layouts/PageTitle/PageTitle';
 import PageBlock from '../layouts/PageBlock';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const InfoPage: React.FC = () => {
```

Comment:
The text/ icons could use some more space between them, it looks cramped rn

Reply:
This might be an issue with ExternalLink component, but I adjusted the vertical spacing between them to make them a bit more spread out

Reply:
Oh woops hold on they are in this component